### PR TITLE
[chiayunrong] iP

### PR DESCRIPTION
**Pull request by ghost**
### Heading

- Bullet list 1
- Bullet list 2

1. Numbered list 1
2. Numbered list 2

```java
public class Main {
    public static void main(String[] args) {
        System.out.println("code block with syntax highlighting");
    }
}
```

- [X] task list 1
- [ ] task list 2

👍 

> block
> quote

[hyperlink](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

`System.out.println("inline code");`

_text_ **formatting** ~~here~~

### CODE REVIEW

File: src/main/java/Alpha.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * The type Alpha.
+ */
```
Comment (chew01) [inline src/main/java/Alpha.java]: Perhaps consider making this class comment more descriptive?

File: src/main/java/Processor.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import com.alpha.commands.Command;
+import com.alpha.exceptions.InvalidTaskException;
+import com.alpha.storage.Storage;
+import com.alpha.tasks.TaskList;
+import com.alpha.ui.Ui;
+import com.alpha.utils.Parser;
+
+/**
+ * The type Processor.
+ */
```
Comment (chew01) [inline src/main/java/Processor.java]: Perhaps consider making this class comment more descriptive?

File: src/main/java/com/alpha/commands/ByeCommand.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        super();
+    }
+
+    /**
+     * Executes the commands.
+     *
+     * @param taskList   Task list of the application.
+     * @param ui      Ui of the application.
+     * @param storage Storage functionality of the application.
+     */
```
Comment (chew01) [inline src/main/java/com/alpha/commands/ByeCommand.java]: Perhaps consider making this method comment more descriptive? What exactly does it do?

File: src/main/java/com/alpha/commands/DeleteCommand.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        super(args);
+    }
+
+    /**
+     * Executes the commands.
+     *
+     * @param taskList   Task list of the application.
+     * @param ui      Ui of the application.
+     * @param storage Storage functionality of the application.
+     */
```
Comment (chew01) [inline src/main/java/com/alpha/commands/DeleteCommand.java]: Perhaps consider making this method comment more descriptive?

**Review by chew01 — COMMENTED** [review]
The code looks good 👍 
Just a few nits, particularly, the class and method comments for command types could be more descriptive.

File: src/main/java/Alpha.java [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+
+/**
+ * The Alpha class.
+ */
+public class Alpha extends Application {
```
Comment (erohsikivar) [inline src/main/java/Alpha.java:22]: You did a great job using different methods to have a better GUI.

File: src/main/java/com/alpha/commands/FindCommand.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+     * @param searchString The search string.
+     * @param taskList     The task list.
+     */
+    public FindCommand(String searchString, TaskList taskList) {
+        super(taskList);
+        this.searchString = searchString;
+    }
+
+    @Override
+    public String execute() {
```
Comment (erohsikivar) [inline src/main/java/com/alpha/commands/FindCommand.java:25]: Good use of String Builder.

File: src/main/java/com/alpha/commands/ToDoCommand.java [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+    private final Task task;
+
+    /**
+     * Instantiates a new To do command.
+     *
+     * @param name    The name of the task.
+     * @param taskList The task list.
+     */
+    public ToDoCommand(String name, TaskList taskList) throws DuplicateTaskException {
```
Comment (erohsikivar) [inline src/main/java/com/alpha/commands/ToDoCommand.java:21]: I never thought of handling duplicate tasks. Good job

**Review by erohsikivar — COMMENTED** [review]
Overall LGTM, good job.

**Pull request closed without merging**